### DISCUSSION TITLE
Error in opening LUSAS BHom Interface

### DISCUSSION THREAD

**Discussion by pathiktank** [BHoM/BHoM#1629] [General] [answered]
Hi I am getting this error while opening LUSAS 19.1 Adapter, has anyone come across this? How to resolve this issue?

![LUSAS Error](https://github.com/user-attachments/assets/3676ed8b-e226-4816-b896-463f066fa829)

**Comment by peterjamesnugent** [comment]
Hi @pathiktank, have you followed the steps in the Wiki?
https://github.com/BHoM/Lusas_Toolkit/wiki/Lusas-Specific-Adapter-Features#starting-the-adapter

**Reply by pathiktank** [reply, marked as answer]
Hi @peterjamesnugent The issue is resolved. I think it does not support rhino 8. When I opened in Rhino7 it worked fine. Thanks for the help
